### ISSUE TITLE
[Issue] Ghasts dissapearing/not rendering to early, before being properly unloaded

### ISSUE THREAD

**Issue by TheonlyTazz** [nanite/ItShallNotTick#12]
### Mod version

ItShallNotTick-1.0.20-build.21.jar

### Forge mod loader version

40.1.69

### Repeatable?

- [X] Yes

### Link to log files

no logs, just visuals

### Steps to reproduce the issue

1. Go to the Nether
2. Find a ghast
3. ghast can dissapear once they are at a certain distance, yet they can still aggro you and shoot you
4. probably die because of random fireballs shooting you

### Description of the issue you are having

ISNT somehow stops the rendering of ghasts while they are still active, meaning they can aggro you and shoot fireballs even if they arent rendered.

putting the Ghast on the entityblacklist fixes this

Also, this happens in Singleplayer with no other players on, even tho the config clearly states
```
	#Minimum number of players before mod is enabled
	minPlayers = 2
```
so maybe thats another issue on its own

This was originally issues on the FTB Modpack Issue tracker:
https://github.com/FTBTeam/FTB-Modpack-Issues/issues

### Information

- [X] I have provided as much information as possible

**Comment by TheonlyTazz** [comment]
Not stale, still waiting for dev to comment on this

**Comment by Gaz492** [comment]
I haven't been able to reproduce this behaviour, do you have a link to the original issue?

**Comment by TheonlyTazz** [comment]
https://github.com/FTBTeam/FTB-Modpack-Issues/issues/1263

i have reproduced this in the latest version of Direwolf20 1.18.2 on a singleplayer world

**Comment by Gaz492** [comment]
Fixed in 67d552a20403100698e49e5ae2e6a112a36c6c2e
